modules/python3/bindings/src/pypickingmapper.cpp

```diff
@@ -89,28 +108,26 @@ void exposePickingMapper(pybind11::module &m) {
         .def_property_readonly("ndc", &PickingEvent::getNDC)
         .def_property_readonly("previousNDC", &PickingEvent::getPreviousNDC)
         .def_property_readonly("pressedNDC", &PickingEvent::getPressedNDC)
+        .def("getWorldSpaceDeltaAtPressDepth", &PickingEvent::getWorldSpaceDeltaAtPressDepth)
         .def_property_readonly("canvasSize", &PickingEvent::getCanvasSize)
         .def_property_readonly("state", &PickingEvent::getState)
         .def_property_readonly("pressState", &PickingEvent::getPressState)
         .def_property_readonly("pressItem", &PickingEvent::getPressItem)
         .def_property_readonly("pressItems", &PickingEvent::getPressItems)
         .def_property_readonly("hoverState", &PickingEvent::getHoverState)
-        .def_property("used", &PickingEvent::hasBeenUsed,
-                      [](PickingEvent *e, bool used) {
-                          if (used) {
-                              e->markAsUsed();
-                          } else {
-                              e->markAsUnused();
-                          }
-                      })
-        .def("markkAsUsed", &PickingEvent::markAsUsed)
-        .def("markkAsUnused", &PickingEvent::markAsUnused)
-        .def("setToolTip", &PickingEvent::setToolTip);
```

Comment:
is there a reason for removing `setToolTip`?

Reply:
ahh was thinking is was in the base... added it again...